packages/web/components/Modals/UnsplashModal/UnsplashModal.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import { createApi } from 'unsplash-js'
+
+import Modal from '@/components/Modal'
+import SearchInput from '@/components/Dashboard/Filters/SearchInput'
+
+type Props = {
+  onCancel: () => void
+  onUnsplashSelect: (smallUrl: string, largeUrl: string) => void
+  show: boolean
+}
+
+type Photo = {
+  id: number
+  width: number
+  height: number
+  urls: { full: string; regular: string; raw: string; small: string; thumb: string }
+  color: string | null
+  user: {
+    username: string
+    name: string
+  }
+}
+
+const unsplashApi = createApi({
```

Comment:
You don't have to do this (although you're welcome to if you'd like), this is mostly a note to self, but I'd like  to dynamically import either this library or or the `UnsplashModal` component itself. The editor page is already one of the heaviest on the site site (which is understandable, the editor involves a lot of code, both from us and third party stuff) but since this is a modal it's a prime candidate to lazy load to cut down on the initial load size of the page.